Base/usr/share/man/man2/dump_backtrace.md

```diff
@@ -0,0 +1,33 @@
+## Name
+
+dump_backtrace - dump the process's current kernel backtrace
+
+## Synopsis
+
+```**c++
+#include <unistd.h>
+
+void dump_backtrace();
+```
+
+## Description
+
+`dump_backtrace` prints the process's kernel backtrace to the serial console. This is most useful for information about a process crash. Note that the userspace backtrace, which in general is more useful, is not printed by this syscall, since it can be printed entirely by userspace examining its own stack.
```

Comment:
nit: `process's` should be `process'`, i think? or reword as `(...) prints the kernel backtrace of a process to the serial console.`

Reply:
According to a quick google, this is only for plurals, so this should be fine as-is. (my spellchecker didn't complain either.)